tests/compilation_tests/robot_test/robot_test.py

```diff
@@ -44,8 +44,10 @@ def test_travel(self):
         new_x = round(-1.3662, 3)
         new_y = round(-0.366198, 3)
         new_theta = round(3 * math.pi / 2, 3)
+        rounded_robot_one_state = [round(state[0], 3)
```

Comment:
why do we round it again after we rounded x, y, and theta?

Reply:
to make the test case work; the previous test case has new_x, new_y, and new_theta be rounded because our travel implementation rounds the state.